xExtension-ReadingTime/static/readingtime.js

```diff
@@ -42,18 +42,20 @@
 
  flux_words_count: function flux_words_count(flux) {
 
-     reading_time.textContent = flux.textContent; // get textContent
+     reading_time.textContent = flux.querySelector('.flux_content').textContent; // get textContent, from the article itself (not the header, not the bottom line). `childNodes[2].childNodes[1]` gives : `<div class="content no_limit">` element
 
      // split the text to count the words correctly (source: http://www.mediacollege.com/internet/javascript/text/count-words.html)
      reading_time.textContent = reading_time.textContent.replace(/(^\s*)|(\s*$)/gi,"");//exclude  start and end white-space
      reading_time.textContent = reading_time.textContent.replace(/[ ]{2,}/gi," ");//2 or more space to 1
      reading_time.textContent = reading_time.textContent.replace(/\n /,"\n"); // exclude newline with a start spacing
 
+     /// This part ↓↓↓ is not working if the content of the title bar is changed by the user… need to do that in proper, more robust way
+     // flux.childNodes[0].childNodes[2].textContent.split(' ').length - // excluding website length
+     // flux.childNodes[0].childNodes[4].textContent.split(' ').length - // excluding date length
+     // flux.childNodes[0].childNodes[6].textContent.split(' ').length;  // excluding title length
+     // Not 100% acurate as it doesn't check if there are shown at the bottom of the article
```

Comment:
No need to explain all of this, since the header is no longer returned by the querySelector